src/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Unix.cs

```diff
@@ -137,6 +137,9 @@ public static SocketError GetSocketErrorForErrorCode(Interop.Error errorCode)
                 case Interop.Error.EHOSTUNREACH:
                     return SocketError.HostUnreachable;
 
+                case Interop.Error.EPERM:
```

Comment:
I don't actually use this anymore now that I have a fallback for this socket error. This is the error you get when trying to open a raw socket without the correct permissions. This seems like a reasonable mapping to me, but we can revert it since I'm not using it anyways.